DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -390,10 +404,20 @@ function Test-TargetResource
         $BindingInfo,
 
         [String[]]
-        $DefaultPage
+        $DefaultPage,
+
+        [String]
+        $EnabledProtocols
     )
 
-    $DesiredConfigurationMatch = $true
+    $PSBoundParameters.GetEnumerator() |
```

Comment:
It was done to Write-Verbose all the bound parameters. Used it for debugging, will remove.